### PR TITLE
Community Article - Setup Cloud Files and ACL (Access Control List): INFODEV-3456

### PR DESCRIPTION
**Pull request by BadCrumpany**

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-78fcaadf70/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-7f80a15dbf/how-to/)

**Comment by RackerKate** [comment]
@cyrichardson Please review.

**Comment by cyrichardson** [comment]
@RackerKate On it.

**Comment by cyrichardson** [comment]
@RackerKate Suggestions here: 
[set-up-cloud-files-and-acls_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2845881/set-up-cloud-files-and-acls_cyrEdits.docx)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-c46b13c7e7/how-to/)

File: content/cloud-files/set-up-cloud-files-and-acls.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+container-level access by using cURL and passing in the following headers:
+
+- `X-Container-Read`: This header holds a comma-delimited list
+  of the users that can read the container. It allows the GET HTTP method for
+  all of the objects in the container.
+- `X-Container-Write`: This header holds a comma-delimited
+  list of users that can write to the container. It allows the PUT, POST, COPY
+  and DELETE HTTP methods for all of the objects in the container.
+
+You can set these special headers only on containers. They apply to all of the
```
Comment (WilliamLoy) [inline content/cloud-files/set-up-cloud-files-and-acls.md]: Suggest: You can set  special headers only on containers, which apply to all of the objects in the container.

File: content/cloud-files/set-up-cloud-files-and-acls.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
+---
+permalink: set-up-cloud-files-and-acls/
+audit_date: '2019-01-22'
```
Comment (WilliamLoy) [inline content/cloud-files/set-up-cloud-files-and-acls.md]: This article appears to have been audited before it was created. I would just make the audit date the same as the created date.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-9840c43f57/how-to/). :bow:

**Pull request merged**